src/java/business/Twit.java

```diff
@@ -15,30 +16,32 @@
  */
 
 public class Twit implements Serializable {
-    private String email, text, date;
+    private long id;
+    private String email, content;
+    private Date posted_date;
     
-    public Twit() {
-        this.email = "";
-        this.date = null;
-        this.text ="";
+    public Twit() {}
+
+    public Twit(String email, String content) {
+        this.email = email;
+        this.content = content;
+        this.posted_date = new Date();
+        this.id = new Random().nextInt(1000000000);
     }
 
-    public Twit(String email, String text) {
+    public Twit(String email, Date posted_date, String content){
+        this.id = new Random().nextInt(1000000000);
```

Comment:
we weren't allowed to have an auto-incremented id so I added a huge random number gen